### PR TITLE
Remove block style registration

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:
We shouldn't be implementing custom block styles in themes as users will lose them when they switch themes. This removes the custom block styles but doesn't remove the CSS, so existing implementations, and the patterns, will still work.

To test, insert patterns that use these styles (e.g. Quadrat > Query Diamond) and check that they still show correctly.

### CODE REVIEW

File: blockbase/inc/block-styles.php [hunk 1]
```diff
@@ -1,64 +0,0 @@
...
-
-	function blockbase_register_block_styles() {
-
-		if ( function_exists( 'register_block_style' ) ) {
-
-			/**
-			 * Register block styles
-			 */
-
-			//Post meta styles with icons
```
Comment (jffng) [inline blockbase/inc/block-styles.php:19]: Just noting that adding the ability to toggle icons to post meta blocks is tracked in GB already:

https://github.com/WordPress/gutenberg/issues/30348
https://github.com/WordPress/gutenberg/issues/24957
https://github.com/WordPress/gutenberg/issues/24956
https://github.com/WordPress/gutenberg/issues/24952

File: seedlet-blocks/inc/block-styles.php [hunk 2]
```diff
@@ -1,38 +0,0 @@
...
-		if ( function_exists( 'register_block_style' ) ) {
-
-			/**
-			 * Register block styles
-			 */
-			register_block_style(
-				'core/latest-posts',
-				array(
-					'name'         => 'seedlet-alternating-grid',
-					'label'        => __( 'Alternating Grid', 'seedlet-blocks' ),
```
Comment (jffng) [inline seedlet-blocks/inc/block-styles.php]: Just confirming we want to remove these, based on @kjellr's reply in slack: p1642598004001100/1642590521.493900-slack-C029FM1EH

**Comment by scruffian** [comment]
I've reverted the change to Seedlet Blocks as we should wait for that to be in Gutenberg before we remove it. I think this is ready for another review.

File: seedlet-blocks/inc/block-styles.php (hunk 2 above)
Comment (pbking) [reply seedlet-blocks/inc/block-styles.php]: Resolved.  Seedlet's stays for now.

**Review by pbking — APPROVED** [review]
This LGTM

🚢

**Pull request merged**